PocketKit/Sources/PocketKit/MyList/SearchItemsList/PremiumExperimentSearch.swift

```diff
@@ -0,0 +1,80 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Sync
+import Combine
+import SharedPocketKit
+
+class PremiumExperimentSearch {
```

Comment:
I'm not sure… I think _for now_ it's best to keep "Experiment" added since _currently_ it's an experiment, and should be renamed later if it's anything otherwise.